web/src/portico/signup.ts

```diff
@@ -295,7 +295,7 @@ $(() => {
         $("#registration .realm-creation-editable-inputs").removeClass("hide");
         $("#id_team_name").trigger("focus");
         // This is a hack to have cursor at end after focussing the input.
-        const name_val = $("#id_team_name").val();
+        const name_val = $("#id_team_name").val()!; // can I add ! here??
```

Comment:
I think you can use non-null assertion here safely because this only returns undefined when called on an empty JQuery collection. I think just to be safe you can also probably use `expectOne` here. So it will be like this -
```typescript
const name_val = $("#id_team_name").expectOne().val()!
```

Reply:
`expectOne()` is a good hint. While `expectOne` and non-null assertion is equivalent here. `expectOne` fails only when the other non-null assertion fails; conversely it's true as well.